main.py

```diff
@@ -62,11 +63,14 @@ def create_ui_components(root):
     update_interval_slider.set(2)
     update_interval_slider.grid(row=3, column=1, padx=10, pady=10, sticky="nsew")
 
-    return transcript_textbox, response_textbox, update_interval_slider, update_interval_slider_label, freeze_button
+    combobox = ctk.CTkOptionMenu(root, values=list(LANGUAGES.values()))                                  
+    combobox.grid(row=3, column=0, padx=200, pady=10, sticky="nsew")                                   
+                                        
+    return transcript_textbox, response_textbox, update_interval_slider, update_interval_slider_label, freeze_button, combobox
 
 def main():
     root = ctk.CTk()
-    transcript_textbox, response_textbox, update_interval_slider, update_interval_slider_label, freeze_button = create_ui_components(root)
+    transcript_textbox, response_textbox, update_interval_slider, update_interval_slider_label, freeze_button, combobox = create_ui_components(root)
```

Comment:
Sry for asking for more changes incrementally but could this new box only appear when we are not in --api mode? Keep in mind good design practices and group this all --api and non --api functionally together.

Reply:
I am new to this so I don't really know what good design practices are but I've tried to group them together.